erigon-lib/state/aggregator.go

```diff
@@ -526,7 +526,23 @@ func (a *Aggregator) BuildMissedAccessors(ctx context.Context, workers int) erro
 		ii.BuildMissedAccessors(ctx, g, ps, missedFilesItems.ii[ii.name])
 	}
 
-	if err := g.Wait(); err != nil {
+	err := func() error {
+		defer func() {
```

Comment:
do we need it?

Reply:
Technically not, same as my other comment.